### DISCUSSION TITLE
Is it possible to be able to have only note objects clickable and not staves and surrounds

### DISCUSSION THREAD

**Discussion by garthholmes** [CoderLine/alphaTab#1770] [Q&A] [answered]
Hello, I would like to be able to block the click action on and around staves, if possible since I have a floating menu that remains in place even when the page is scrolling that I can make disappear by clicking on the page..

**Comment by Danielku15** [comment, marked as answer]
Hi. 
No, this is currently not possible out-of-the-box. You would need to fully opt-out of the built-in click handling via [`enableuserinteraction`](https://www.alphatab.net/docs/reference/settings/player/enableuserinteraction#description) and then build your own logic how you wanna handle clicks. You can look at the original click handling code for reference: https://github.com/CoderLine/alphaTab/blob/develop/src/AlphaTabApiBase.ts#L1251

Alternatively you could register a [capture event listener ](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#usecapture) on a higher level DOM element and [stop event propagation](https://developer.mozilla.org/en-US/docs/Web/API/Event#instance_methods) to alphaTab itself based on a custom condition.
